packages/toolpad-app/src/theme.ts

```diff
@@ -712,18 +712,13 @@ export function getThemedComponents(theme: Theme): { components: Theme['componen
         defaultProps: {
```

Comment:
Some minor changes to the theme to make the search input look aligned with the rest of the UI